lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -55,6 +59,7 @@ def compute(self, student, course_structure, scores_client, submissions_scores):
             # self.scores may hold outdated data, force it to refresh on next access
             lazy.invalidate(self, 'scores')
 
+        log.info("accessed lazy property scores with value: {0}".format(unicode(self.scores)))
```

Comment:
I agree we ought to catch this type of thing in unit tests, but this issue has bitten us once already. Could we leave the "computed" and "accessed" log statements in for our manual testing on edge, and remove them before full rollout?